package.json

```diff
@@ -112,6 +110,10 @@
     "webpack-stream": "3.2.0",
     "cross-env": "^3.1.4"
   },
+  "peerDependencies": {
+    "react": "^0.14.0 || ^15.0.0",
```

Comment:
Right now we require `15.4.2` I guess we shouldn't still support `0.14.0` then? 🤔